executor/merge_join_test.go

```diff
@@ -806,10 +824,7 @@ func (s *testSuiteJoin3) TestVectorizedMergeJoin(c *C) {
 		{[]int{chunkSize - 1}, []int{chunkSize - 1}},
 		{[]int{chunkSize - 1}, []int{chunkSize + 1}},
 		{[]int{chunkSize}, []int{chunkSize}},
-		{[]int{chunkSize}, []int{chunkSize - 1}},
 		{[]int{chunkSize}, []int{chunkSize + 1}},
-		{[]int{chunkSize + 1}, []int{chunkSize}},
```

Comment:
Why remove these cases?

Reply:
According to L823,  it will test twice for one test case. So the test case `{[]int{chunkSize + 1}, []int{chunkSize}}` is reduplicated with  `{[]int{chunkSize}, []int{chunkSize + 1}}`.  We only need to keep one of those.